### PR TITLE
Test PR 2

### PR DESCRIPTION
**Pull request by nedtwigg**

### CODE REVIEW

**Comment by nedtwigg** [comment]
This PR has no comment, fair enough.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ Toolbox for the Modeling and Analysis of Thermodynamic Systems <br>
 For questions, comments, and general support see the <b> <a href= "https://groups.google.com/forum/#!forum/t-mats-user-group" >T-MATS user's forum</a></b>. <br>
 Please take a moment and fill out our <b> <a href= "https://docs.google.com/forms/d/1cjcCyOKZpV49-gsdGGaKHUsZ2c2fKX5mwkKrntt60Eo/viewform?usp=send_form" >Survey</a></b>, user interest and feedback allows us to continue working on this package.<br><br>
 
-<b>Introduction</b> <br>
```
Comment (nedtwigg) [inline README.md:11]: Single comment review.

**Comment by edgartwigg** [comment]
A comment from me.

File: .diffplug-pr/pr_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+### `path/to/thingy`
+
+![](https://raw.githubusercontent.com/diffplug/T-MATS/baaffcfbfc5258121e0bebabb406305d8f6a92f3/.diffplug-pr/Screen%20Shot%202020-07-20%20at%201.15.46%20PM.png)
```
Comment (edgartwigg) [inline .diffplug-pr/pr_2.md:4]: I think this might be a problem.

File: README.md [hunk 3]
```diff
@@ -8,7 +8,6 @@ Toolbox for the Modeling and Analysis of Thermodynamic Systems <br>
...
 The Toolbox for the Modeling and Analysis of Thermodynamic Systems (T-MATS) 
 is a Simulink toolbox intended for use in the modeling and simulation of thermodynamic 
 systems and their controls. T-MATS contains generic thermodynamic and controls
```
Comment (edgartwigg) [inline README.md:13]: And I'm not a fan of this either.

**Review by edgartwigg — CHANGES_REQUESTED** [review]
Not great.

File: .diffplug-pr/pr_2.md (hunk 2 above)
Comment (nedtwigg) [reply .diffplug-pr/pr_2.md:4]: I dunno, looks okay to me.

**Comment by nedtwigg** [comment]
Apologies, just doing testing, didn't mean to open this on upstream.

**Pull request closed without merging**